Active Documents/contentEditableDisabled.html

```diff
@@ -7,84 +7,168 @@
+        </p>
+        <h3>
+            <code>contenteditabledisabled</code> Allowed Tokens.
+        </h3>
+        <p>
+            <ul>
+                <li><code>formatjustifycenter</code></li>
+                <li><code>formatjustifyfull</code></li>
+                <li><code>formatjustifyleft</code></li>
+                <li><code>formatjustifyright</code></li>
+                <li><code>formatbackcolor</code></li>
+                <li><code>formatbold</code></li>
+                <li><code>insertlink</code></li>
+                <li><code>formatfontname</code></li>
+                <li><code>formatfontsize</code></li>
+                <li><code>formatfontcolor</code></li>
+                <li><code>formatindent</code></li>
+                <li><code>inserthorizontalrule</code></li>
+                <li><code>insertorderedlist</code></li>
+                <li><code>insertunorderedlist</code></li>
+                <li><code>formatitalic</code></li>
+                <li><code>formatoutdent</code></li>
+                <li><code>formatremove</code></li>
+                <li><code>formatstrikethrough</code></li>
+                <li><code>formatsubscript</code></li>
+                <li><code>formatsuperscript</code></li>
+                <li><code>formatunderline</code></li>
+            </ul>
+        </p>
+        <p class="issue" title="missing fontSize and insertImage">
+            fontSize / insertImage is missing from beforeInput.  Why?  It seems that it should be listed to create a more complete set.
+            <a href="https://github.com/w3c/editing/issues/249">Issue 249</a>
+        </p>
+        <h3>
+            <code>contenteditabledisabled</code> Allowed Token Shortcuts.
+        </h3>
+        <p>
+            The following tokens are also supported by the <code>contenteditabledisabled</code> attribute and serve as shorthands for specifying a set of tokens from contenteditabledisabled Allowed Tokens:
+        </p>
+        <p>
+            <ul>
+                <li><code>formatjustify</code>: combines <code>formatjustifycenter</code>, <code>formatjustifyfull</code>, <code>formatjustifyleft</code>, and <code>formatjustifyright</code>.</li>
+                <li><code>formatfont</code>: combines <code>formatfontname</code>, <code>formatfontsize</code>, and <code>formatfontcolor</code></li>
+                <li><code>formatdent</code>: combines <code>formatindent</code> and <code>formatoutdent</code></li>
+                <li><code>insertlist</code>: combines <code>insertorderedlist</code> and <code>insertunorderedlist</code></li>
+                <li><code>formatscript</code>: combines <code>formatsubscript</code> and <code>formatsuperscript</code></li>
+            </ul>
+        </p>
+        <p class="note">
+            Note that this specification does not have a shortcut to disable all editing UI.  Authors should consider using a <code>textarea</code> or <code>contenteditable="plaintext-only"</code> for this purpose.
```

Comment:
I think `plaintext-only` shouldn't be recommended as it's currently removed from the spec.